### ISSUE TITLE
The Magical Code Injection Rainbow

### ISSUE THREAD

**Issue by andresriancho** [andresriancho/w3af#8694]
SpiderLabs has created a test web application:
https://github.com/SpiderLabs/MCIR

It might be a good idea to:
 - [ ] Review the test cases to understand if they are real or CTF-like
 - [ ] Write unittests which scan this application and find vulnerabilities
  - [ ] eval: https://github.com/SpiderLabs/MCIR/tree/master/phpwn
  - [ ] RFI: https://github.com/SpiderLabs/MCIR/tree/master/rfidk
  - [ ] OS commanding: https://github.com/SpiderLabs/MCIR/tree/master/shellol
  - [ ] SQL injection (might be difficult, requires mysql): https://github.com/SpiderLabs/MCIR/tree/master/sqlol
  - [ ] XML injection: https://github.com/SpiderLabs/MCIR/tree/master/xmlmao
  - [ ] XSS: https://github.com/SpiderLabs/MCIR/tree/master/xssmh

## Related with
Write unittests against all WAVSEP sections #1800

**Comment by andresriancho** [comment]
https://github.com/SpiderLabs/MCIR/pull/4

**Comment by unicornsasfuel** [comment]
SQLol supports SQLite if you change the database config file at sqlol/includes/database.config.php. A commented-out example SQLite configuration section is provided.

**Comment by andresriancho** [comment]
@unicornsasfuel yes, the setup will be easier, but there are some tests which require a real DB (SLEEP delays) and some payloads only work in mysql; others only in pgsql, etc.

Best case would be to have different test applications, one for each DB.
